ncm-authconfig/src/main/pan/components/authconfig/schema.pan

```diff
@@ -17,7 +17,7 @@ type yesnostring = string with match(SELF, "yes|no");
 
 type authconfig_pamadditions_line_type = {
   "order"       : string with match(SELF, '^(first|last)$')
-  "entry"       : string with match(SELF, '\S+\.so$')
+  "entry"       : string with match(SELF, '^\s*\w+\s+\S+\.so(\s|$)')
```

Comment:
Is this unnecessarily restrictive? how about 
'^\s*\w+\s+\S+\.so'

\w+ could possibly be restricted to the set of words (e.g. (required|sufficient|etc) allowed in that position in the pam file?